less/index.less

```diff
@@ -57,3 +57,176 @@ html, body {
     height: 100%;
 }
 
+/* My code */
+
+::selection{
+	background-color: #C8C8C8;
+}
+
+@desktopRootFontH1: 2rem;
+@centerDivVertically: 0 auto;
+@tableTextAlignment:center;
+
+.container{
+	width: 90%;
+	margin: @centerDivVertically;
+	border: 3px solid #C8C8C8;
+	margin-top: 3rem;
+}
+
+.page-navigation{
+
+	padding: 20px;
+	
+	ul{
+		display: flex;
+		justify-content: flex-end;
+	}
+	ul li{
+		display: inline;
+		margin-right:20px;
+
+		a{
+			color: black;
+			text-decoration: none;
+
+			&:hover{
+				text-decoration: line-through;
+				text-decoration-color: grey;
+			}
+		}
+	}
+}
+
+@media (max-width:500px) and (max-width:800px){
+	.page-navigation{
+		width: 100%;
+	}
+}
+
+
+.intro{
+	padding: 2rem;
+	background-color: #C8C8C8;
+
+	h1{
+		font-size: @desktopRootFontH1;
+		padding: 1rem;
+	}
+
+	p{
+		padding: 1rem;
+	}
+
+	img{
+		width: 30%;
+		height: 30%;
+	}
+	
+	button{
+		margin:1rem;
+		padding: 0.5rem;
+		background: grey;
+		color: white;
+		border: 1px solid grey;
+	}
+}
+
+
+.about-me{
+	height: 20rem;
+	display: flex;
+	padding: 2rem 0;
+	justify-content: space-between;
+	text-align: center;
+	align-items: center;
+	margin: 2rem 0;
+
+	.about-me-text{
+		width: 50%;
+		padding:2rem;
+
+		h1{
+			font-size: @desktopRootFontH1; 
+			text-align: left;
+			padding: 1rem 0;
+		}
+	}
+
+	.skills{
+		width: 45%;
+		padding: 2rem;
+	}
+}
+
+.about-me-text p{
+	line-height: 2rem;
```

Comment:
You might have set 2rem as a variable also since you used it multiple times in your code